content/english/Kusto-KC7/activity-0.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Activity 0: Login & Setup of the Cyber Environment"
+description: ""
+draft: false
+weight: 4
+---
+
+{{< youtube VL7fJr54CG0 >}}
```

Comment:
Use something similar to this for all youtube videos:

```suggestion
<p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/VL7fJr54CG0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
```

Reply:
Done. I think the aspect ratio needs to be bigger on the page though, we can discuss